libraries/AP_Scripting/lua_scripts.cpp

```diff
@@ -213,8 +220,11 @@ void lua_scripts::create_sandbox(lua_State *L) {
     lua_pushstring(L, "utf8");
     luaopen_utf8(L);
     lua_settable(L, -3);
-    load_generated_sandbox(L);
+    lua_pushstring(L, "package");
+    luaopen_package(L);
```

Comment:
https://github.com/ArduPilot/ardupilot/pull/23354#discussion_r1161892699 could set path here instead.

Reply:
The default path also adds a bunch of different places to look. I suspect we would be better with just a single path to avoid confusion, just `scripts/modules/?.lua;scripts/modules/?/init.lua`

Reply:
its member of package module. it needs to happen while opening the lib. I have removed the root path.